matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/internal/DefaultPatientMatch.java

```diff
@@ -21,9 +21,10 @@
 import org.phenotips.data.Patient;
 import org.phenotips.data.PatientRepository;
 import org.phenotips.data.similarity.PatientSimilarityView;
+import org.phenotips.data.similarity.phenotype.DefaultPhenotypesMap;
```

Comment:
How does\did this work, `DefaultPhenotypesMap` seems to have been used before but not imported?